compiler_opt/es/blackbox_optimizers.py

```diff
@@ -180,7 +175,7 @@ def __init__(self,
     # Check step_size and ga_optimizer
     if bool(step_size) == bool(ga_optimizer):
```

Comment:
if (step_size is None) == (ga_optimizer is None)

because bool(step_size)=False, and someone may feed step_size=0 to debug.

Reply:
The commit has been amended to reflect this